pex/pep425tags.py

```diff
@@ -271,10 +271,10 @@ def get_supported(versions=None, noarch=False, platform=None,
         abis[0:0] = [abi]
 
     abi3s = set()
-    import imp
-    for suffix in imp.get_suffixes():
-        if suffix[0].startswith('.abi'):
-            abi3s.add(suffix[0].split('.', 2)[1])
+    import importlib
```

Comment:
@jsirois any reason this is a local import ?
can we move it to the top of the file?